frontend/src/core/l10n/components/AppLocalizationProvider.js

```diff
@@ -47,7 +47,9 @@ export class AppLocalizationProviderBase extends React.Component<InternalProps>
     render() {
         const { children, l10n } = this.props;
 
-        return <LocalizationProvider bundles={ l10n.bundles }>
+        const localizations = new ReactLocalization(l10n.bundles);
```

Comment:
My intent in the new API was that `ReactLocalization` instances are stored in the app's state, because I've seen issues with storing just the `bundles` (esp. when it's a generator rather than an array, which doesn't apply in Pontoon's case).

I suggest to make the `core/l10n.actions:get` action return an instance of `ReactLocalization` instead of `bundles`. You won't have to instantiate it in `AppLocalizationProvider.render` then.